### PR TITLE
Add configuration for Ratis retry cache expiry time

### PR DESCRIPTION
**Pull request by bf8086**
Add a configuration for Ratis retry cache expiry time. This affects the size of the Ratis server retry cache, which can consume a lot of memory if there is a high volume of journal writes in a short period of time.

### CODE REVIEW

**Comment by bf8086** [comment]
@gpang @LuQQiu PTAL.

File: core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java [hunk 1]
```diff
@@ -324,6 +324,12 @@ private synchronized void initServer() throws IOException {
                 .getMs(PropertyKey.MASTER_EMBEDDED_JOURNAL_TRANSPORT_REQUEST_TIMEOUT_MS),
             TimeUnit.MILLISECONDS));
 
+    RaftServerConfigKeys.RetryCache.setExpiryTime(properties,
+        TimeDuration.valueOf(
+            ServerConfiguration.global()
```
Comment (gpang) [inline core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java:329]: Can this not be `ServerConfiguration.getMs()`?

**Review by gpang — COMMENTED** [review]
@bf8086 just a minor comment

File: core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java (hunk 1 above)
Comment (bf8086) [reply core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java]: Good to know. Updated.

**Comment by bf8086** [comment]
Thanks @gpang! I updated the PR. PTAL.

**Review by gpang — APPROVED** [review]
LGTM (after tests pass)

**Comment by bf8086** [comment]
alluxio-bot, merge this please.

**Comment by bf8086** [comment]
alluxio-bot, cherry-pick to branch-2.4 please.

**Comment by bf8086** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please.

**Pull request merged**